roles/openshift_version/tasks/main.yml

```diff
@@ -1,206 +1,2 @@
-  # - may contain numbers
-  # - may containe dots (https://github.com/openshift/openshift-ansible/issues/5192)
-  #
-  - name: (Enterprise) Verify openshift_image_tag is valid
-    when: openshift.common.deployment_type == 'openshift-enterprise'
-    assert:
-      that:
-      - "{{ openshift_image_tag is match('(^v\\d+\\.\\d+(\\.\\d+)*(-\\d+(\\.\\d+)*)?$)') }}"
-      msg: |-
-        openshift_image_tag must be in the format v#.#[.#[.#]]. Examples: v1.2, v3.4.1, v3.5.1.3,
-        v3.5.1.3.4, v1.2-1, v1.2.3-4, v1.2.3-4.5, v1.2.3-4.5.6
-        You specified openshift_image_tag={{ openshift_image_tag }}
-
-# Make sure we copy this to a fact if given a var:
-- set_fact:
-    openshift_version: "{{ openshift_version | string }}"
-  when: openshift_version is defined
-
-# Protect the installed version by default unless explicitly told not to, or given an
-# openshift_version already.
-- name: Use openshift.common.version fact as version to configure if already installed
-  set_fact:
-    openshift_version: "{{ openshift.common.version }}"
-  when:
-  - openshift.common.version is defined
-  - openshift_version is not defined or openshift_version == ""
-  - openshift_protect_installed_version | bool
-
-# The rest of these tasks should only execute on
-# masters and nodes as we can verify they have subscriptions
-- when:
-  - inventory_hostname in groups['oo_masters_to_config'] or inventory_hostname in groups['oo_nodes_to_config']
-  block:
-  - name: Set openshift_version for rpm installation
-    include_tasks: set_version_rpm.yml
-    when: not openshift_is_containerized | bool
-
-  - name: Set openshift_version for containerized installation
-    include_tasks: set_version_containerized.yml
-    when: openshift_is_containerized | bool
-
-  - block:
-    - name: Get available {{ openshift_service_type}} version
-      repoquery:
-        name: "{{ openshift_service_type}}"
-        ignore_excluders: true
-      register: rpm_results
-    - fail:
-        msg: "Package {{ openshift_service_type}} not found"
-      when: not rpm_results.results.package_found
-    - set_fact:
-        openshift_rpm_version: "{{ rpm_results.results.versions.available_versions.0 | default('0.0', True) }}"
-    - name: Fail if rpm version and docker image version are different
-      fail:
-        msg: "OCP rpm version {{ openshift_rpm_version }} is different from OCP image version {{ openshift_version }}"
-      # Both versions have the same string representation
-      when:
-      - openshift_rpm_version != openshift_version
-      # if openshift_pkg_version or openshift_image_tag is defined, user gives a permission the rpm and docker image versions can differ
-      - openshift_pkg_version is not defined
-      - openshift_image_tag is not defined
-    when:
-    - openshift_is_containerized | bool
-    - not openshift_is_atomic | bool
-
-  # Warn if the user has provided an openshift_image_tag but is not doing a containerized install
```

Comment:
This check adds exactly 0 value.  Nobody is going to see this random debug statement in the sea of output unless they are specifically looking for it.  In which case, they probably already know.